exp-fashion-mnist/hyperparams.json

```
@@ -0,0 +1,31 @@
+{
+    "adagrad": {
+        "dwell": 20,
+        "initial_batch_size": 128,
```

```suggestion
        "initial_batch_size": 256,
```

Then that enables a good comparison between AdaGrad and RadaDamp-LR.